contrib/skopeo/template.py

```diff
@@ -0,0 +1,49 @@
+pkgname = "skopeo"
+pkgver = "1.6.0"
+pkgrel = 0
+build_style = "go"
+# for compatibility with Makefile targets
+make_dir = "bin"
```

Comment:
```suggestion
```

not used

Reply:
Used to create `bin/skopeo` which Makefile expects.

Reply:
nothing that you wrote here makes the makefile 'expect' this. the docs or install- targets you call don't depend on that.

Reply:
in podman where you copied this from this is actually used because the podman install: target is called, which is not the case here

Reply:
(and with the addition of the tests below, this does get used, but it was not the case before that)

Reply:
They do expect this. See the `completions` comment above. Please don't say I'm wrong when you clearly haven't checked for yourself...

Reply:
it is true in 1.16 indeed